### ISSUE TITLE
WEEK1-Quiz

### ISSUE THREAD

**Issue by Ashley0324** [Ashley0324/Cousera-Neural-Networks-and-Deep-Learning#1]
**Q1：Which of the following best describes the role of AI in the expression "an AI-powered society"?**
AI is an essential ingredient in realizing tasks, in industry and in personal life.

**Q2：Which of these are reasons for Deep Learning recently taking off? (Check the three options that apply.)**
Deep learning has resulted in significant improvements in important applications such as online advertising, speech recognition, and image recognition.
We have access to a lot more computational power.
We have access to a lot more data.

**Q3：Recall the diagram of iterating over different ML ideas. Which of the stages shown in the diagram was improved with the use of a better GPU/CPU?**
Some algorithms are specifically designed to run experiments faster.

**Q4： Neural networks are good at figuring out functions relating an input xx with an output yy given enough examples. True/False?**
True.

**Q5: Which of the following depicts a Sigmoid activation function?**
image

**Q6:Images for cat recognition is an example of “structured” data because it is represented as a structured array in a computer. True/False?**
False.

**Q7: A dataset is composed of age and weight data for several people. This dataset is an example of "structured" data because it is represented as an array in a computer. True/False?**
True

**Q8: Why is an RNN (Recurrent Neural Network) used for machine translation, say translating English to French? (Check all that apply.)**
It can be trained as a supervised learning problem.
It is applicable when the input/output is a sequence (e.g., a sequence of words).

**Q9: Suppose the information given in the diagram is accurate. We can deduce that when using large training sets, for a model to keep improving as the amount of data for training grows, the size of the neural network must grow. True/False?.**
True

**Q10: Assuming the trends described in the figure are accurate. The performance of a NN depends only on the size of the NN. True/False?**
False
